client/src/components/ManageAccount/tests/ManageAccountModals.test.js

```diff
@@ -348,6 +389,48 @@ describe('Edit Required Information', () => {
 
     expect(getByText(/Submit/)).toBeDefined();
   });
+  it('should render a form with this phone number schema', () => {
```

Comment:
@nathanfranklin  Currently there is no restriction on submitting a phone format on TAS. But this is a custom restriction on the frontend side that I think numbers that are valid for when you update your account information. Also since 'should render a form' and 'should show users errors if the fields are missing or invalid' is basically testing what it need to be I deleted the test codes.